.github/workflows/main.yml

```diff
@@ -54,6 +54,9 @@ jobs:
   test-oss:
     runs-on: ubuntu-22.04
     needs: build-oss-for-test
+    strategy:
+      matrix:
+        path_style: [virtual, virtual-v2]
```

Comment:
**Annotation:**
We are only testing both path styles at the main test as a compromise on test time.  This is the primary place where something might be different and the rest of tests are variations on deployment options.